src/reynir_correct/tools/diffchecker.py

```diff
@@ -44,15 +44,15 @@ def main():
     # Hægt að biðja um annað til að fá frekari upplýsingar!
     options["format"] = "text"  # text, json, csv, m2
     options["annotations"] = True
-
     options["all_errors"] = True
-
     # options["infile"] = open("prufa.txt", "r")
-    options["one_sent"] = True
+    options["one_sent"] = False
     # options["generate_suggestion_list"] = True
-    options["ignore_comments"] = False
-    # options["annotate_unparsed_sentences"] = False
-
+    options["ignore_comments"] = True
+    options["annotate_unparsed_sentences"] = True
+    options["ignore_wordlist"] = set(["flargor"])
```

Comment:
What's this - a test case left over?

Reply:
Yes, Deleted now. The input I tested with:
`# Test á ignore_wordlist í options
Ég var flargor í gær.
`